e2e: remove the generic ephemeral volume support check for kube version

humblec opened the pull request:
at present we have the check for kube version 1.21 in the tests
which no longer required as it falls under supported kubernetes
versions with the driver.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on e2e/README.md:
```diff
@@ -111,7 +111,7 @@ are available while running tests:
 you need to follow these steps before running e2e.
 
-- Install snapshot controller and Beta snapshot CRD
+- Install snapshot controller and snapshot CRD
```
@humblec this change is nothing to do with this PR?

humblec replied:
true.. may be I will put it as a seperate commit

Madhu-1 commented on e2e/cephfs.go:
```diff
@@ -298,35 +298,32 @@ var _ = Describe(cephfsType, func() {
 			}
 			By("verify generic ephemeral volume support", func() {
-				// generic ephemeral volume support is beta since v1.21.
-				if k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
```
same changes are required in rbd.go also.

humblec replied:
sure.. will club it in the same PR

humblec committed:
e2e: remove the generic ephemeral volume support check

at present we have the check for kube version 1.21 in the tests
which no longer required as it falls under supported kubernetes
versions with the driver.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
e2e: remove snapshot beta CRD mention from the doc

The snapshot CRD is no longer Beta and this commit adjust the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
@Madhu-1 comments are addressed. ptal.. thanks.

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.23

humblec commented:
@Mergifyio refresh
